statusHandler/termuic/termuiRenders/widgetsRender.go

```diff
@@ -60,14 +64,16 @@ func (wr *WidgetsRender) setGrid() {
 	gridLeft := ui.NewGrid()
 
 	gridLeft.Set(
-		ui.NewRow(10.0/22, wr.instanceInfo),
-		ui.NewRow(12.0/22, wr.chainInfo))
+		ui.NewRow(5.0/11, wr.instanceInfo),
```

Comment:
I liked it more with 22, because that was the actual height in rows. But it can remain like this.

Reply:
Changed with 22 because is more clear.